codec-native-quic/src/main/c/netty_quic_boringssl.c

```diff
@@ -597,12 +612,59 @@ void keylog_callback(const SSL* ssl, const char* line) {
     (*e)->CallVoidMethod(e, keylogCallback, keylogCallbackMethod, (jlong) ssl, keyString);
 }
 
-static jlong netty_boringssl_SSLContext_new0(JNIEnv* env, jclass clazz, jboolean server, jbyteArray alpn_protos, jobject handshakeCompleteCallback, jobject certificateCallback, jobject verifyCallback, jobject servernameCallback, jobject keylogCallback, jint verifyMode, jobjectArray subjectNames) {
+int new_session_callback(SSL *ssl, SSL_SESSION *session) {
```

Comment:
Add a link to https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#SSL_CTX_sess_set_new_cb and explain why we return 0 (as we take no ownership)